src/components/MapView/MapView.web.tsx

```diff
@@ -34,7 +105,7 @@ const MapView = forwardRef<MapViewHandle, MapViewProps>(
             if (waypoints.length === 1) {
                 mapRef.flyTo({
                     center: waypoints[0].coordinate,
-                    zoom: 15,
+                    zoom: CONST.MAPBOX.DEFAULT_ZOOM,
```

Comment:
Comment for reference

Reply:
Just a heads up, this has caused an issue in https://github.com/Expensify/App/issues/32261

The zoom level was changed from 15 to 10 for web, but it's still 15 for other platforms. 
This caused an issue when the default zoom and zoom when you select a location is the same, so if the default location and the location you've selected are the same - the map isn't zoomed in
This was changed back to 15 in https://github.com/Expensify/App/pull/33234